python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/ethernet_interfaces.schema.yml

```
@@ -2267,9 +2267,22 @@ keys:
                 type: str
                 convert_types:
                   - int
+            srlgs:
+              description: List of SRLGs.
+              type: list
+              items:
+                type: str
+                description: SRLG name or number.
+                convert_types:
+                  - int
             srlg:
               description: SRLG name or number.
               type: str
+              deprecation:
+                warning: true
+                removed: false
+                remove_in_version: 6.0.0
+                new_key: traffic_engineering.srlgs
```

```suggestion
                new_key: srlgs
```
Reason for having only `srlgs` as new key is it will through the error if both old and new key is defined in your input data. Applicable for port-channel interfaces too.

---

Thanks, have applied this to both eth and po interface schemas!